add xpp cryo chamber screen + lakeshore 336 widget

vespos opened the pull request:
New custom XPP screen for the cryo chamber setup.

ZLLentz commented on widgets/motor_small.ui:
```diff
@@ -106,14 +106,44 @@
         <string>1</string>
        </property>
+       <property name="releaseValue" stdset="0">
+        <string>None</string>
+       </property>
        <property name="relativeChange" stdset="0">
-        <bool>true</bool>
+        <bool>false</bool>
+       </property>
+       <property name="writeWhenRelease" stdset="0">
+        <bool>false</bool>
```
Do you know what these small changes to motor_small.ui were for?

vespos replied:
Yes, for some reason that widget was set to relative change the TWF or TWR parameter. This led to the motor not moving the right amount.
I guess the other parameters became "alive" from the window I edited this in...

ZLLentz commented on xpp_screens/cryo_chamber.sh:
```diff
@@ -0,0 +1,12 @@
+SCRIPT=${SCRIPT%%.*}
+
+echo "Launching ${SCRIPT}.ui"
+pydm --hide-nav-bar $MYDIR/$SCRIPT.ui
```
Here's my screenshot:
![image](https://user-images.githubusercontent.com/10647860/222008565-ab3762c8-115d-4d77-b9e3-95b5f6f8a4e2.png)

Looks totally reasonable :+1:

Since there are a bunch of motors in a row here, it's making me think about motor row widgets and what the optimal ordering would be. I didn't expect EGU first, for example, I usually expect the EGU after the readback but maybe my assumptions are wrong in the face of usage testing.

vespos replied:
I agree, the order of the things could be revised here. I was wondering the same think about whether we want to have setpoint or readback first for example.

ZLLentz commented on widgets/LS336.ui:
```diff
@@ -0,0 +1,415 @@
+        <property name="text">
+         <string>Temps control
+Lakeshore 336
+(Not implemented)</string>
```
Are you planning to expand this to do something later?

vespos replied:
It is implemented now that we figured out the shell script issue. I have a new commit for that (commenting below about it)

vespos commented:
Implemented the link to the ls336 edm controls screen.
That also made me realized that custom shell script are not convenient to run unless the script that launches the script also cd into the top level of the package. Added that command for all existing screens launch script for completeness

ZLLentz commented:
I'll re-review tomorrow 👍

ZLLentz commented:
It still looks good and the button works now :+1:
The only thought I have is that the scrollbar might be placed incorrectly, since if you scroll down you hide the upper motors. Maybe it should be only on the lakeshore side? Maybe each half should have it's own scroll that only appears as needed? But then again, this only matters for very specific ranges of screen sizes.

![image](https://user-images.githubusercontent.com/10647860/222267207-13062359-511e-4689-b28e-23ecd8109c74.png)

vespos commented:
I think that when run on the CR computers, the screen will be full size and the scroll bar is not needed. It's really just a convenience for when the screen is opened on a smaller screen.
